### ISSUE TITLE
[changelog] Existing `CHANGELOG.md` file not found when running in Gradle via IntelliJ or in subproject

### ISSUE THREAD

**Issue by gotson** [jreleaser/jreleaser#1145]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. configure `changelog.append.enabled=true`
2. `gradle jreleaserChangelog`

### Expected Behaviour

Changelog should be appended to existing `CHANGELOG.md`.

### Actual Behaviour

Crash.

In the `trace.log` i can see this:

```
[WARN]  Can not read changelog from ../../../.gradle/daemon/7.5.1/CHANGELOG.md
```

This seems weird, it seems JReleaser is trying to resolve the path of the existing `CHANGELOG.md` from the current directory where the process is run, instead of using the project root directory.

If i run it from the command line instead, it also crashes, but i have a different warning and error:

```
[WARN]  Can not read changelog from ../CHANGELOG.md
```

Note that JReleaser is configured on a sub-project, but i have also configured `gitRootSearch` to `true`.

### Environment Information

- **Operating System**: macOS
- **JReleaser Version:** 1.4.0
- **JDK Version:** 11

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

This is the log when running from IntelliJ:

[trace.log](https://github.com/jreleaser/jreleaser/files/10349712/trace.log)

This is the log when running from the command line in the project root:

[trace.log](https://github.com/jreleaser/jreleaser/files/10349719/trace.log)

**Comment by gotson** [comment]
A workaround is to explicitly set the target path: `target.set(rootDir.resolve("CHANGELOG.md"))`

**Comment by aalmiray** [comment]
Released in v1.5.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.5.0
